roles/remove-node/pre-remove/tasks/main.yml

```diff
@@ -16,7 +16,7 @@
       --ignore-daemonsets
       --grace-period {{ drain_grace_period }}
       --timeout {{ drain_timeout }}
-      --delete-local-data {{ item }}
+      --delete-local-data {{ kube_override_hostname|default(item) }}
```

Comment:
How does this work with a loop on inventory ?

Reply:
Good catch. It seems broken. I tested on single node setup so that went through. Let me push a fix. :+1:

Reply:
@floryut I have updated the PR. Can you please have a second look? :)